app/views/users/show.html.erb

```diff
@@ -1,2 +1,14 @@
-<h1>This is a user</h1>
+<div class="users_container">
+<%= render partial: 'shared/card_user', locals: {user: @user} %>
+<p class="user_bio"><%= @user.bio%></p>
+<%#Add post for the user%>
+<% @posts.each do |post| %>
+<a href="<%=post.author_id%>/posts" class="user_link">
+<%= render partial: 'shared/card_post', locals: {post: post} %>
+</a>
+<%end%>
+<div class="btn"><button type="button" class="see_all_post">See all post</button></div>
+</div>
+
```

Comment:
- [ ] Please see the button should send the user to the page where he can see all the posts you implemented on the page but not the link.
![image](https://user-images.githubusercontent.com/74744787/216397008-449bcd17-3204-42fe-81ff-95002f7ab78a.png)
![image](https://user-images.githubusercontent.com/74744787/216397899-4a6ef1d2-ef2f-4960-8574-0ee77f73c680.png)